<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="ID"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="MobilityModel1">
         <item>
          <property name="text">
           <string>RandomWalker</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>standstill</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HorizontalWalker</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VerticalWalker</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WalkerStartAtEast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WalkerStartAtNorth</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WalkerStartAtSouth</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WalkerStartAtWest</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CSBehaviorMode1">
         <item>
          <property name="text">
           <string>PowerOnPowerOff</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>poweron</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PowerOnOffPsCnt</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PowerOnOffPsCntRecover</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PowerOnForcePowerOff</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PowerOnPowerOffEmergency</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PowerOnEmergency</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PowerOnOffPsCntEmergency</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PowerOnOffPsCntRecoverEmergency</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PowerOnForcePowerOffEmergency</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="PSBehaviorMode1">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="UEType1">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>IratHO</string>
         </property>
         <item>
          <property name="text">
           <string>IratHO</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tet_Area1"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Pairs name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>Amount of pairs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mobility Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>CS Behavior Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>PS Behavior Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>UE Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Area</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
